### PR TITLE
MBS-8795 Update FlakyAnnotation.md

### PR DESCRIPTION
**Pull request by sergioboi**
add onSdks description

### CODE REVIEW

File: docs/content/docs/test/FlakyAnnotation.md [hunk 1]
```diff
@@ -28,15 +28,18 @@ We want:
...
 
 ```kotlin
-@Flaky(reason = "Relies on real back-end")
+@Flaky(reason = "Relies on real back-end", onSdks = [22, 28])
 class MyAwesomeTests {
     // ...
 }
 ```
 
+- `reason` - optional description of flakiness. You will see that reason in ReportViewer
```
Comment (eugene-krivobokov) [inline docs/content/docs/test/FlakyAnnotation.md]:
```suggestion
- `reason` - description of flakiness. You will see this reason in ReportViewer
```
I propose to omit optionality in order to discourage null values.

File: docs/content/docs/test/Runner.md [hunk 2]
```diff
@@ -143,7 +143,7 @@ In our case:
 
 Check out a configuration to run in `GradleInstrumentationPluginConfiguration` in the [test app](https://github.com/avito-tech/avito-android/blob/develop/samples/test-app/build.gradle.kts#L114)
 
-## Filtering tests for execution
+## Filtering tests from execution
```
Comment (eugene-krivobokov) [inline docs/content/docs/test/Runner.md]: Are you sure?
We filter tests before execution. Execution is the purpose of filtering. 
https://wikidiff.com/content/vs-whats-difference-33

Comment (sergioboi) [reply docs/content/docs/test/Runner.md]: No

Comment (sergioboi) [reply docs/content/docs/test/Runner.md]:
```suggestion
## Filtering tests for execution
```

**Review by eugene-krivobokov — APPROVED** [review]

**Pull request merged**